rust/blockstore/src/types.rs

```diff
@@ -86,7 +86,7 @@ impl Value for Vec<u32> {
 
 impl Value for &[u32] {
     fn get_size(&self) -> usize {
-        self.len() * size_of::<u32>()
+        std::mem::size_of_val(*self)
```

Comment:
My read is that `&[u32]` is a dynamically sized val, so that's what we'll get.  What do you think?  https://doc.rust-lang.org/std/mem/fn.size_of_val.html